core/examples/http_bench_ops.rs

```diff
@@ -217,10 +217,7 @@ fn main() {
 
   let future = async move {
     js_runtime
-      .execute(
-        "http_bench_json_ops.js",
-        include_str!("http_bench_json_ops.js"),
-      )
+      .execute("http_bench_ops.js", include_str!("http_bench_ops.js"))
       .unwrap();
```

Comment:
Since this example is being renamed, maybe we should also use "tcp" instead of "http" in the name?

This would make it correspond better with [deno_tcp.ts](https://github.com/denoland/deno/blob/9d53dab4df64361f00dda2534eecbd04797d56a2/cli/bench/deno_tcp.ts)

Reply:
Makes sense, we could also call it `htcp` to indicate that it's not fully http but not unstructured tcp either.

Reply:
Reverted this name change, it's somewhat unimportant to this PR and can be changed later. Changing this has the undesirable side-effect of making benchmarks (on our benchmarks page) harder to follow.